src/index.html

```diff
@@ -15,7 +15,7 @@
     <meta content="#ffffff" name="theme-color">
     <link href="assets/images/favicon.ico?" rel="shortcut icon" type="image/x-icon">
     <title>
-        Coastal Resilience – Mapping portal
+        Coastal Resilience – Mapping portal
```

Comment:
Possibly trivial but I can't tell what changed in this line. Different whitespace char?

Reply:
Yeah, just some whitespace.